### DISCUSSION TITLE
Translation - documentation

### DISCUSSION THREAD

**Discussion by shuther** [nextify-limited/saasfly#91] [General]
I was wondering if there was particular steps to translate and add a language. Are you using crowdin or something else? Would you have steps where/what I should change to add/remove languages?

**Comment by jackc3** [comment]
Thank you for your feedback. We are considering open sourcing this repository, so stay tuned.

**Reply by mateusloubach** [reply]
I'd like to contribute to translation/language as well. are you accepting contributions or do you already have someone for that?

**Reply by jackc3** [reply]
Hi @mateusloubach , thanks for this. We were considered about accepting contributions. But we are working on the next version. We will start preparing it after releasing the next major version.

**Reply by mateusloubach** [reply]
Hey @jackc3 , thanks for the reply. That's great to hear. I'll be on the lookout for when you make it available.

**Comment by shuther** [comment]
if that helps, you can create a crowdin.yaml file like below:
```
#
# Your Crowdin credentials
#
"project_id_env": "CROWDIN_PROJECT_ID"
"api_token_env": "CROWDIN_PERSONAL_TOKEN"
"base_path": "."
"base_url": "https://api.crowdin.com"

#
# Choose file structure in Crowdin
# e.g. true or false
#
"preserve_hierarchy": true

#
# Files configuration
#
files: [
  {
    #
    # Source files filter
    # e.g. "/resources/en/*.json"
    #
    "source": "/apps/nextjs/src/config/dictionaries/en.json",

    #
    # Where translations will be placed
    # e.g. "/resources/%two_letters_code%/%original_file_name%"
    #
    "translation": "/apps/nextjs/src/config/dictionaries/%two_letters_code%.json",

    #
    # Files or directories for ignore
    # e.g. ["/**/?.txt", "/**/[0-9].txt", "/**/*\?*.txt"]
    #
    # "ignore": [],

  }
]
```
